governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Inclusion Poll - Implementation of an Antalpha Institutional Vault - May 23, 2022
```

Comment:
```suggestion
title: Investigate Implementation of an Antalpha Institutional Vault - May 23, 2022
```

Reply:
Probably wouldn't bother with "Inclusion Poll" 

Add "Investigate" to the poll file name too @Hipogeo 

Also need to add a space " " after the word "Vault" in the file name